rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala

```diff
@@ -70,17 +71,43 @@ trait PersistentStoreTester {
       dbDir.recursivelyDelete()
     }
   }
+
+  def fixture[R](f: (RhoISpace[Task], ChargingReducer[Task]) => Task[R])(
+      implicit errorLog: ErrorLog[Task]
+  ): R = {
+    implicit val logF: Log[Task]           = new Log.NOPLog[Task]
+    implicit val metricsEff: Metrics[Task] = new metrics.Metrics.MetricsNOP[Task]
+    mkRhoISpace[Task]("rholang-interpreter-test-")
+      .use { rspace =>
+        for {
+          cost <- CostAccounting.emptyCost[Task]
+          reducer = {
+            implicit val c = cost
+            RholangOnlyDispatcher.create[Task, Task.Par](rspace)._2
+          }
+          _   <- reducer.setPhlo(Cost.UNSAFE_MAX)
+          res <- f(rspace, reducer)
+        } yield res
+      }
+      .runSyncUnsafe(3.seconds)
+  }
 }
 
 class ReduceSpec extends FlatSpec with Matchers with PersistentStoreTester {
   implicit val rand: Blake2b512Random = Blake2b512Random(Array.empty[Byte])
 
+  /**
+    Do not use this method, see explanation below.
+    Use mapData and assert at call site
+    */
   def checkData(
       result: Map[
         Seq[Par],
         Row[BindPattern, ListParWithRandom, TaggedContinuation]
       ]
   )(channel: Par, data: Seq[Par], rand: Blake2b512Random): Assertion =
+    // The 'should' assertion will fail on this line making it harder to find
+    // the exact spot where the error manifested itself
```

Comment:
Your newly introduced tests don't differ in their shape from the existing ones. Please either:

1. fix all the tests w.r.t. to checkData usage and remove the method
2. extract a common core from the tests, fix the `checkData` issue once
   (either write your tests in the old style first and refactor all, or refactor and then add your tests in the new style)

In any case, do retain consistency, so that others can later do further cleanup more easily,

Reply:
This test class is huge so 1) is going to make this PR explode. Better if this is done in a separate PR.
2a) is not possible due to how `checkData` is currently written. 
Looks to me that 2b) = 1)

Reply:
>  make this PR explode

not really, and even if - it's fine as long as you keep the commits fine-grained.

> done in a separate PR.

I'm fine with that if you do that immediately after merging this. Otherwise it'd be better if you kept the new tests consistent with the (granted, poor) style the other tests have.

Reply:
> I'm fine with that if you do that immediately after merging this.

Yeah, can do
> Otherwise it'd be better if you kept the new tests consistent with the (granted, poor) style the other tests have.

I cannot use `checkData` in it's current form, so that's not possible